src/families/cardano/hw-getAddress.ts

```diff
@@ -0,0 +1,40 @@
+import type { Resolver } from "../../hw/getAddress/types";
+import Ada, {
+  Networks,
+  AddressType,
+} from "@cardano-foundation/ledgerjs-hw-app-cardano";
+import { str_to_path } from "@cardano-foundation/ledgerjs-hw-app-cardano/dist/utils";
+import { getBipPathFromString, getBipPathString } from "./logic";
+import { StakeChain } from "./types";
+import { CARDANO_NETWORK_ID, STAKING_ADDRESS_INDEX } from "./constants";
+import { utils as TyphonUtils } from "@stricahq/typhonjs";
+
+const resolver: Resolver = async (transport, { path }) => {
+  const spendingPath = getBipPathFromString(path);
+  const stakingPathString = getBipPathString({
+    account: spendingPath.account,
+    chain: StakeChain.stake,
+    index: STAKING_ADDRESS_INDEX,
+  });
+  const ada = new Ada(transport);
+  const r = await ada.deriveAddress({
```

Comment:
On the issue of displaying the address or not, here's maybe what you can do:
The `Resolver` type has a `GetAddressOptions` param that itself has a `verify` param: https://github.com/LedgerHQ/ledger-live-common/blob/5fa6a917da4a4789d757ae67eb0679f54ce6cd52/src/hw/getAddress/types.ts#L14
This param is already provided as needed by the callers of receive (live desktop and live mobile).

Also from what I can see the ledgerjs-hw-app-cardano helper provides a [showAddress](https://github.com/cardano-foundation/ledgerjs-hw-app-cardano/blob/6ec2cc3fa869ef1ee1e6033792aae80c49962141/src/Ada.ts#L282) function in addition to [deriveAddress](https://github.com/cardano-foundation/ledgerjs-hw-app-cardano/blob/6ec2cc3fa869ef1ee1e6033792aae80c49962141/src/Ada.ts#L265).

So maybe here you can use the `verify` param to just call the appropriate helper function?
I'm just guessing here, to be confirmed. :slightly_smiling_face:

Reply:
yup, this is exactly what I did after our last meeting and it is working all good :)